taqueria-vscode-extension/package.json

```diff
@@ -145,11 +152,44 @@
 				"enablement": "@taqueria/plugin-flextesa"
 			},
 			{
-				"command": "taqueria.deploy",
+				"command": "taqueria.originate",
 				"category": "Taqueria",
-				"title": "Deploy contracts",
-				"shortTitle": "deploy",
+				"title": "Originate contracts",
+				"shortTitle": "originate",
 				"enablement": "@taqueria/plugin-taquito"
+			},
+			{
+				"command": "taqueria.opt_in",
+				"category": "Taqueria",
+				"title": "Opt-in to analytics",
+				"shortTitle": "opt-in"
```

Comment:
If it's possible to toggle between opt-in/opt-out based on the current setting that would be ideal

For example, if the user is opted-in to analytics, hide the opt-in task as it's redundant

Reply:
Currently, TVsCE runs `taq` after scaffolding. But because projects are outdated, it does nothing. So once we upgrade those, this problem should be fixed.

Reply:
Let me see if I can watch that file outside the workspace and enable/disable opt[in/out].
By the way, if that file does not exist (or if the setting is missing from the file), do we consider the user as opted in or out?

Reply:
If the file or property does not exist, the user is considered `opted-out`